Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -815,18 +801,26 @@
                                                             VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
                                                             ZoomMode="{TemplateBinding ScrollViewer.ZoomMode}">
                                                             <Grid>
+                                                                <Grid.RowDefinitions>
+                                                                    <RowDefinition Height="*" />
+                                                                    <RowDefinition Height="Auto" />
+                                                                </Grid.RowDefinitions>
                                                                 <Grid.ColumnDefinitions>
                                                                     <ColumnDefinition Width="Auto" />
                                                                     <ColumnDefinition Width="*" MinWidth="24" />
                                                                 </Grid.ColumnDefinitions>
                                                                 <ItemsPresenter
                                                                     Padding="{TemplateBinding Padding}"
                                                                     HorizontalAlignment="Stretch"
-                                                                    Footer="{TemplateBinding Footer}"
                                                                     FooterTemplate="{TemplateBinding FooterTemplate}"
                                                                     FooterTransitions="{TemplateBinding FooterTransitions}"
                                                                     HeaderTemplate="{TemplateBinding HeaderTemplate}"
                                                                     HeaderTransitions="{TemplateBinding HeaderTransitions}" />
+
+                                                                <ContentPresenter
```

Comment:
Why the change? The footer now appears left aligned instead of center-aligned, was it intentional?

Reply:
Should be fixed in 649e004